### PR TITLE
Added CHANGELOG.md Check

### PR DESCRIPTION
**Pull request by ivanglushko**

### CODE REVIEW

**Review by ivanglushko — COMMENTED** [review]
🤠 Please update the **CHANGELOG.md** file.
> Keep going sir, next time do not break the rules 🫡

**Comment by ivanglushko** [comment]
Please update the CHANGELOG.md file.

**Comment by ivanglushko** [comment]
🤠 Please update the **CHANGELOG.md** file.
  > Keep on going sir, next time do not break the rules 🫡

**Review by akhaman — APPROVED** [review]
😍

**Review by PravosudovTinkoff — APPROVED** [review]

File: .github/workflows/check-changelog.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+name: Check CHANGELOG.md has been written.
+
+on:
+  workflow_call:
+    secrets:
```
Comment (ivanglushko) [inline .github/workflows/check-changelog.yml:5]: 1

**Pull request merged**